### ISSUE TITLE
Respect "show bundle" setting for bundles

### ISSUE THREAD

**Issue by Ajedi32** [joeyespo/inbox-by-gmail-checker#1]
Inbox provides a feature that lets you hide bundles from the inbox and only show them once every day or week. (So you can deal with all the emails in the bundle at once, instead of dealing with them individually as they come in.)

This extension should respect that setting, and not include those emails in the count shown on the Inbox icon until they are displayed in the inbox.

**Comment by Ajedi32** [comment]
Looks like this would be pretty difficult, as it seems this extension is currently getting the unread count from Gmail, not Inbox.

**Comment by joeyespo** [comment]
Yeah, this would be hard with the way it's set up. I think you'd have to have an official "Sign in with Google" to get access to the right API. (Please correct me if I'm wrong.)

Thanks for the issue. It's definitely worth adding if there's a good way to do it.

**Comment by joeyespo** [comment]
This issues seems to come up a lot in [the reviews](https://chrome.google.com/webstore/detail/inbox-by-gmail-checker/mpjmeeikbbgccbjkbfabocnjcaejdpmj/reviews).

Anyone have any ideas on how to fix this without requiring an additional login?

**Comment by Ajedi32** [comment]
Well, the extension already has [permission to read from `inbox.google.com` directly](https://github.com/joeyespo/inbox-by-gmail-checker/blob/master/manifest.json#L23). So maybe that would be enough?

Anyway, I think most extensions like this that I've seen don't require a separate login, just an oauth token. (The user has to click through a permissions prompt from Google before the extension will work properly.) What API were you planning to use?
